src/kudos/guards/slackToken.guard.ts

```diff
@@ -0,0 +1,29 @@
+import {Observable} from "rxjs";
+import {CanActivate, ExecutionContext, Injectable} from "@nestjs/common";
+import {SlackService} from "../services/slack.service";
+
+@Injectable()
+export class SlackTokenGuard implements CanActivate {
+
+    constructor(private slackService: SlackService) {
+    }
+
+    canActivate(context: ExecutionContext): boolean | Promise<boolean> | Observable<boolean> {
+        const request = context.switchToHttp().getRequest();
+        const timeWhenResponseUrlIsAvailable = new Date().getTime() + 3001
```

Comment:
`new Date().getTime() + 3001` pojawia się w kilku miejscach. Warto byłoby to wyciągnąć do jednego, zwłaszcza, że `3000 + 1` nie jest oczywistą wartością.